### PR TITLE
[greynoise-266] Add second pack for GreyNoise Community (free) API

### PR DESCRIPTION
**Pull request by bradchiappetta**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Description
This is a spin-off of the GreyNoise pack, but is specifically for our Community (free) users and contains just a single action to allow for the GreyNoise Community API to be queried.

## Screenshots
image

## Minimum version of Cortex XSOAR
- [X]  5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [X] Tests
- [X] Documentation

### CODE REVIEW

File: Packs/GreyNoise_Community/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+##### What does this pack do?
+This pack allows for analysts to integrate the free, GreyNoise Community API data into their playbooks and workflows.
+It gives them the ability to determine if an IPv4 address is something they need to worry about or is just
+Internet-Background noise.
+
+- Allows analysts to query an IP from GreyNoise and determine if it is Internet-Background Noise.
+- Allows analysts to query an IP from GreyNoise and determine if it belongs to a benign service or provider.
+
+
+_For more information, visit our [GreyNoise Developer Documentation](https://developer.greynoise.io)_
```
Comment (mchasepan) [inline Packs/GreyNoise_Community/README.md:14]: @bradchiappetta Suggest add a link for them to get more information on your site on upgrading to the full API

Comment (mchasepan) [reply Packs/GreyNoise_Community/README.md:14]: And differences between this API/pack and the full one.

File: Packs/GreyNoise_Community/pack_metadata.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "name": "GreyNoise Community",
+    "description": "GreyNoise is a threat intelligence service that collects and analyzes Internet-wide scan and attack traffic. With this integration, users can access the free, GreyNoise Community API to provide enrichment data to their workflows. The full integration code can be found here: https://github.com/demisto/content/tree/master/Packs/GreyNoise_Community",
+    "support": "partner",
+    "currentVersion": "1.0.0",
+    "author": "GreyNoise",
+    "url": "https://greynoise.io",
+    "email": "hello@greynoise.io",
```
Comment (mchasepan) [inline Packs/GreyNoise_Community/pack_metadata.json:8]: @bradchiappetta the other pack points to "sales@greynoise.io" which address should customers email for help/support?

Comment (bradchiappetta) [reply Packs/GreyNoise_Community/pack_metadata.json:8]: @mchasepan : they should actually probably point to support@greynoise.io.  I'll update this one shortly, and the other pack when I push the ReadMe update.

File: Packs/GreyNoise_Community/Integrations/GreyNoise_Community/GreyNoise_Community.py [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+    demisto.debug(f"Command being called is {demisto.command()}")
+    try:
+
+        client = Client(
+            api_key=api_key,
+            api_server=api_server,
+            timeout=10,
+            proxy=handle_proxy("proxy", proxy).get("https", ""),
+            use_cache=False,
+            integration_name=f"xsoar-community-integration-v{demisto.demistoVersion()['version']}",
```
Comment (Itay4) [inline Packs/GreyNoise_Community/Integrations/GreyNoise_Community/GreyNoise_Community.py:291]:
```suggestion
            integration_name=f"xsoar-community-integration-v{get_demisto_version_as_str()}",
```

**Review by Itay4 — COMMENTED** [review]
Looks good 👍 
1. I think it makes more sense to add this integration to the existing content pack instead of the creating a new one
2. The `Community` terminology conflicts with the marketplace, can we use another word for it?

**Comment by bradchiappetta** [comment]
> Looks good 👍
> 
> 1. I think it makes more sense to add this integration to the existing content pack instead of the creating a new one
> 2. The `Community` terminology conflicts with the marketplace, can we use another word for it?

@Itay4: we discussed this with @mchasepan and decided that with the way the GreyNoise Community is established, it works best to have this as an independent pack, rather than potentially having unusable/confusing additional actions (since everything in the current pack requires a paid subscription) in the pack that they couldn't use.  We would prefer to continue to refer to this as "GreyNoise Community" since it is specifically for GreyNoise Community users who use the GreyNoise Community API.  I'm open to other suggestions on how we can best fit this in to not confuse users on either side.

**Comment by sangsoojun88** [comment]
Everything looks good on my end

**Comment by bradchiappetta** [comment]
@Itay4 : I see you added a "pending-demo" tag here.  Can you clarify what you need on this?

**Comment by Itay4** [comment]
a session with @ShahafBenYakir will be scheduled by @mchasepan

**Comment by bradchiappetta** [comment]
Closing this PR in favor of adding these commands into the GreyNoise Pack, but as a different Integration set per conversation with @ShahafBenYakir @mchasepan

**Pull request closed without merging**